deploy/docker/porter/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3'
+
+services:
+  porter-http:
+    restart: on-failure
+    image: nucypher:latest
+    build:
+      context: ../../..
+      dockerfile: deploy/docker/Dockerfile
+    ports:
+      # Default Porter port
+      - "80:9155"
+    volumes:
+      - .:/code
+      - ~/.local/share/nucypher:/nucypher
+    command: ["nucypher", "porter", "run",
+              "--provider", "${WEB3_PROVIDER_URI}",
+              "--network", "${NUCYPHER_NETWORK}"]
+
+  porter-https:
+    restart: on-failure
+    image: nucypher:latest
+    build:
+      context: ../../..
+      dockerfile: deploy/docker/Dockerfile
+    ports:
+      # Default Porter port
+      - "443:9155"
+    volumes:
+      - .:/code
+      - ~/.local/share/nucypher:/nucypher
+      - "${TLS_DIR}:/etc/porter-tls/"
+    command: [ "nucypher", "porter", "run",
+               "--provider", "${WEB3_PROVIDER_URI}",
+               "--network", "${NUCYPHER_NETWORK}",
+               "--tls-key-filepath", "/etc/porter-tls/key.pem",
+               "--tls-certificate-filepath", "/etc/porter-tls/cert.pem"]
```

Comment:
Essentially, you expect that users will modify these paths to match their own system?

local TLS termination is a nice feature here.

Reply:
There is an associated volume mapping on L32.

```
"${TLS_DIR}:/etc/porter-tls/"
```

so my thought was that users would create a folder with `key.pem` and `cert.pem` files and then set the `TLS_DIR` environment variable to be that folder.

Reply:
Ah - I see it now.  Thanks for clarifying.

Reply:
👍